### ISSUE TITLE
Accessibility JS module is missing for Highcharts

### ISSUE THREAD

**Issue by dsathyakumar** [eBay/ebayui-core#1904]
# Bug Report

## Description
Accessibility JS module is missing for Highcharts.
In the `versions.json` file, the entry for this file appears to be missed.

## Workaround
We could pass the options `cdnHighchartsAccessibility` in the `input` to the component.

**Comment by agliga** [comment]
This is not needed. The accessibility module is using the same version as highcharts. https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-line-chart/component.js#L31-L33
